PocketBeagle/pru/Makefile

```diff
@@ -0,0 +1 @@
+include /var/lib/cloud9/common/Makefile
```

Comment:
Is this needed if we are using the runner? I'd rather the runner use -f. Needing this file is confusing.

Reply:
Looking, it looks like you left my runner, but renamed it. or, maybe I forgot to push. Anyway, I guess you added this to make things run easier from the CLI and I'm cool with that. ;-)

Reply:
Yes, I renamed c.run to C (make).run and things started working.  I don't know why, but it works.

I like have the cli option too.